### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by elmigranto**
Updated some libs, fixed linting, "archived" old code, cleaned html a bit.

### CODE REVIEW

**Comment by j3k0** [comment]
Quick notice (I know it's work in progress, but since I get notifications by email), make sure you can build the docker image.

See https://hub.docker.com/r/ganomede/admin/builds/ for last automated build's error report.

```
Step 14 : COPY web/templates /home/app/code/web/templates
lstat web/templates: no such file or directory
```

**Comment by elmigranto** [comment]
Can we bump node.js to 5.10 so it switches to npm v3?

**Comment by j3k0** [comment]
Yes sure.

**Comment by j3k0** [comment]
You have to change `circle.yml` and `Dockerfile` to apply the change everywhere.

File: Dockerfile [hunk 1]
```diff
@@ -8,34 +7,19 @@ WORKDIR /home/app/code
...
-		js/entrypoint.js > bundle.js)
-
-COPY web/images /home/app/code/web/images
-COPY web/img /home/app/code/web/img
-COPY web/pics /home/app/code/web/pics
-COPY web/css /home/app/code/web/css
-
-COPY server /home/app/code/server
+# npm install, etc.
+RUN (cd /home/app/code && make)
```
Comment (j3k0) [inline Dockerfile]: Bad approach: `npm install` will have to be re-run each time any file is changed.

Previous code allowed to only run it when package.json was changed.

Secondly, copying `server` after running browserify allows not to re-run browser when unrelated server side code is changed.

Lastly, `make` doesn't need to be installed on production servers... That's why I run browserify directly.

Comment (elmigranto) [inline Dockerfile]: so we should?

```
copy package.json && npm install
copy web && browserify
copy server, index.js, config.js
```

**Comment by elmigranto** [comment]
Okay, this one builds with no errors and seems to address all the issues you mentioned.

**Comment by elmigranto** [comment]
Okay, let's merge if `Dockerfile` looks good to you.

**Pull request merged**